react-accordion-component/src/accordion.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+
+class Accordion extends React.Component {
+  constructor(props) {
+    super(props);
+    this.props = props;
+    this.state = {
+      target: null,
+      isTargetOn: false
```

Comment:
You can change this state to one property `index: null`. If the index of an item matches this state, then it should be showing.